packages/skeleton/src/lib/utilities/TableOfContents/TableOfContents.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+	import { fade } from 'svelte/transition';
+
+	// Types
+	import type { CssClasses } from '../../index.js';
+	// Stores
+	import { tocStore, tocActiveId } from './stores.js';
+
+	// Props (styles)
+	/** Provide text for the title. Accepts HTML. */
+	export let title = 'Table of Contents';
```

Comment:
Passing HTML as a string here doesn't feel very svelty. We should either only accept regular text or provide a named slot if they want to add their own HTML.

Reply:
Agreed, we'll make it a slot.